docs_user/modules/proc_migrating-ceph-mds.adoc

```diff
@@ -2,28 +2,33 @@
 
 = Migrating {Ceph} MDS to new nodes within the existing cluster
 
-In the context of data plane adoption, where the {rhos_prev_long} ({OpenStackShort}) services are
-redeployed in {rhocp_long}, a {OpenStackPreviousInstaller}-deployed {CephCluster} cluster will undergo a migration in a process we are calling “externalizing” the {CephCluster} cluster.
-There are two deployment topologies, broadly, that include an “internal” {CephCluster} cluster today: one is where {OpenStackShort} includes dedicated {CephCluster} nodes to host object storage daemons (OSDs), and the other is Hyperconverged Infrastructure (HCI) where Compute nodes double up as {CephCluster} nodes. In either scenario, there are some {Ceph} processes that are deployed on {OpenStackShort} Controller nodes: {Ceph} monitors, Ceph Object Gateway (RGW), Rados Block Device (RBD), Ceph Metadata Server (MDS), Ceph Dashboard, and NFS Ganesha.
-This document describes how to migrate the MDS daemon in case {rhos_component_storage_file_first_ref} (deployed with either a cephfs-native or ceph-nfs backend) is part of the overcloud deployment. The MDS migration is performed by cephadm, and as done for the other daemons, the general idea is to move the daemons placement from a "hosts" based approach to a "label" based one.
+During the data plane adoption, where the {rhos_prev_long} ({OpenStackShort}) services are redeployed in {rhocp_long}, a {OpenStackPreviousInstaller}-deployed {CephCluster} cluster is migrated in a process called externalizing the {Ceph} cluster.
+
+There are two deployment topologies that currently include an internal {CephCluster} cluster: 
+
+* {OpenStackShort} includes dedicated {CephCluster} nodes to host object storage daemons (OSDs). 
+* Hyperconverged Infrastructure (HCI), where Compute nodes double up as {CephCluster} nodes. 
+
+In either scenario, there are some {Ceph} processes that are deployed on {OpenStackShort} Controller nodes: {Ceph} monitors, Ceph Object Gateway (RGW), Rados Block Device (RBD), Ceph Metadata Server (MDS), Ceph Dashboard, and NFS Ganesha.
```

Comment:
I think L5 to L12 is more a general introduction. It represents a leftover because initially we wrote these procedures as standalone documents. Now that we connected everything together, I think this introduction can go away entirely.